pyaedt/edb_core/EDB_Data.py

```diff
@@ -587,6 +589,50 @@ def filling_material_name(self, value):
             self._filling_material_name = value
             self.update_layers()
 
+    @property
+    def negative_layer(self):
+        """Negative layer.
+
+        Returns
+        -------
+        bool
+            True if the layer is negative, else False.
```

Comment:
```suggestion
            ``True`` when negative, ``False`` otherwise.
```

Reply:
If the only option other than negative is positive, change to ``True`` when negative, ``False`` when positive.

Reply:
Done